### PR TITLE
Add cru-udp-pipeline job

### PR DESCRIPTION
**Pull request by Omicron7**
Adds project which will be used to deploy lambda functions to the snowplow sub-account.

### CODE REVIEW

File: jobs/pipeline-jobs.yml [hunk 1]
```diff
@@ -51,6 +51,13 @@
     jobs:
       - 'pipeline-template':
 
+- project:
+    name: cru-udp-pipeline
+    description: |
+      Cru User Data Platform pipeline (serverless) function.
+    jobs:
+    - 'pipeline-template':
+        branches: master
```
Comment (mattdrees) [inline jobs/pipeline-jobs.yml]: you don't want to have a staging env?

Comment (Omicron7) [reply jobs/pipeline-jobs.yml]: We don't have a staging snowplow pipeline anymore.

**Comment by Omicron7** [comment]
I tested the assumeRole code on jenkins-lab in an ssh session, and confirmed it generates the proper files. I also tested the string interpolation in a groovy console to make sure it produces valid output for bash.

**Comment by mattdrees** [comment]
maybe squash those last two commits

**Comment by mattdrees** [comment]
are all the details worked out? worth squashing/merging this now?

**Comment by Omicron7** [comment]
Yes, this is good to go. I squashed all the random commits already. The remaining 3 each did a specific change.

**Comment by mattdrees** [comment]
>  I squashed all the random commits already.

Sorry, I should have checked more closely.

**Pull request merged**